en-US/focus-ios.xliff

```diff
@@ -1,776 +1,749 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 http://docs.oasis-open.org/xliff/v1.2/os/xliff-core-1.2-strict.xsd">
-  <file original="Blockzilla/Info.plist" source-language="en" datatype="plaintext" target-language="en">
+  <file original="Blockzilla/en.lproj/Find in page done.strings" datatype="plaintext" source-language="en">
```

Comment:
@sblatz unclear what happened here - this is an export that I did today, and this extra string is added now. The one string in this table, `FindInPage.Done` should simply be in the `Localizable.strings`.